FirebaseAuth/Sources/Backend/RPC/FIRVerifyAssertionRequest.h

```diff
@@ -96,6 +96,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, assign) BOOL autoCreate;
 
+/** @property displayName
+    @brief A displayName linked to its provider
+ */
+@property(nonatomic, copy, nullable) NSString *displayName;
```

Comment:
You'll need to make changes in `FIRVerifyAssertionRequest.m` as well.

Reply:
Thanks, Have made changes